boot/zephyr/Kconfig

```diff
@@ -197,6 +197,7 @@ config BOOT_VALIDATE_SLOT0_ONCE
 config BOOT_PREFER_SWAP_MOVE
 	bool "Prefer the newer swap move algorithm"
 	default y if SOC_FAMILY_NRF
+	default !$(dt_nodelabel_enabled,scratch_partition)
 	default n
```

Comment:
Does this line have to stay after the new one?

Reply:
I slightly updated the line and removed the `default n`.
In case a scratch_partition is available it will implicitly be set to `default n` and could theoretically be overwritten later on.

https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults